src/userobjects/MoabSkinner.C

```diff
@@ -215,9 +220,25 @@ MoabSkinner::getAuxiliaryVariableNumber(const std::string & name,
   return _fe_problem.getAuxiliarySystem().getFieldVariable<Real>(0, name).number();
 }
 
+MooseMesh &
+MoabSkinner::getMooseMesh()
+{
+  if (_use_displaced)
+  {
+    return _fe_problem.getDisplacedProblem()->mesh();
+  }
+  else
+  {
+    return _fe_problem.mesh();
+  }
+}
 MeshBase &
 MoabSkinner::mesh()
```

Comment:
Are the `getMooseMesh()` and `mesh()` functions here doing basically the same thing? Can we consolidate them into a single function call instead?